### PR TITLE
query and update animals table

### PR DESCRIPTION
**Pull request by ismailakinkunmi**
- insert data
- use transaction
- write queries to answer questions

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -5,4 +5,48 @@ SELECT date_of_birth FROM animals WHERE name IN('Agumon','Pikachu');
...
+update animals set species ='digimon' where name like '%mon';
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+update animals set species ='pokemon' where species is null;
+commit;
+
+--  Inside a transaction delete all records in the animals table, then roll back the transaction
+begin;
+delete from animals *;
+rollback;
+
```
Comment (keneogu) [inline queries.sql:27]: Inside a transaction, Add

- [ ] `DELETE` all animals born after Jan 1st, 2022 and create a savepoint for the transaction

- [ ]  `UPDATE` all animals' weight to be their weight multiplied by -1. and  Rollback to the savepoint

- [ ] `UPDATE` all animals' weights that are negative to be their weight multiplied by -1. and Commit transaction.

File: queries.sql [hunk 2]
```diff
@@ -5,4 +5,48 @@ SELECT date_of_birth FROM animals WHERE name IN('Agumon','Pikachu');
...
+----How many animals are there?
+select * from animals;
+
+----How many animals have never tried to escape?
+select * from animals where escape_attempts = 0;
+
+---- What is the average weight of animals?
+-15.5500000000000000
+
+----Who escapes the most, neutered or not neutered animals?
```
Comment (keneogu) [inline queries.sql:57]: Create a query for who escapes the most, neutered or not neutered animals?

File: queries.sql [hunk 3]
```diff
@@ -5,4 +5,48 @@ SELECT date_of_birth FROM animals WHERE name IN('Agumon','Pikachu');
...
+Boarmon
+
+----What is the minimum and maximum weight of each type of animal?
```
Comment (keneogu) [inline queries.sql:60]: Create a query for the minimum and maximum weight of each type of animal?

File: queries.sql [hunk 4]
```diff
@@ -5,4 +5,48 @@ SELECT date_of_birth FROM animals WHERE name IN('Agumon','Pikachu');
...
+ species | min | max
+---------+-----+------
+ pokemon | -17 |  -11
+ digimon | -45 | -5.7
+
+ ----What is the average number of escape attempts per animal type of those born between 1990 and 2000?
```
Comment (keneogu) [inline queries.sql:66]: Create a query for this too

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @ismailakinkunmi, 

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history
- Good PR description

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [ ] Your Repo has no scheme.sql, the scheme.sql should contain the new column called `species` with a string type.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Review by omar-labana — APPROVED** [review]
# **Approved ✔️**

Hi @ismailakinkunmi 

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**